.eslintrc.js

```diff
@@ -12,9 +12,18 @@ module.exports = {
   rules: {
     'import/newline-after-import': 1,
     'import/order': ["error", { "newlines-between": "always", "groups": ["builtin", ["external", "internal"], "parent", "sibling", "index"] }],
-    'prettier/prettier': ['error'],
-    'sort-imports': 'off', // No auto-fix!
+    'prettier/prettier': [
```

Comment:
Can we move this to `.prettierrc`?

Reply:
yea, all information about prettier config can be in eslintrc :) (1 config file less. Everything is in one place <3)

Reply:
Yes, but some people might like to use the prettier plugin in their respective dev env. It's fine for the root of our repo to consist of only config files I think.. that's the way of jsdev nowadays

Reply:
ok, sure